packages/worker/src/worker-options.ts

```diff
@@ -252,6 +255,12 @@ export interface WorkerOptions {
   debugMode?: boolean;
 
   bundlerOptions?: {
+    /**
+     * Before we bundle the Workflow code with Webpack, we call `configureWebpack`, passing the Webpack
```

Comment:
Don't use "we" in API reference, say "Before Workflow code is bundled with Webpack, ..."